tagger.py

```diff
@@ -0,0 +1,66 @@
+import django
+import os
+
+# Need to set up django app first
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'data_portal.settings')
+django.setup()
+
+# ---
+
+import logging
+import sys
+from datetime import datetime
+
+from data_processors import services
+from utils import libs3
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+log_file = logging.FileHandler("tagger-{}.log".format(datetime.now().strftime("%Y%m%d%H%M%S")))
+log_file.setLevel(logging.INFO)
+log_file.setFormatter(logging.Formatter("%(asctime)s %(name)-12s %(levelname)-8s %(message)s"))
+logger.addHandler(log_file)
+
+
+FILTER_SUFFIX = ".bam"
+
+if __name__ == '__main__':
+    """
+    Meant to run one time, offline tool to re/process tagging as implemented in services.tag_s3_object(..)
+    
+    Cost:
+    1. Listing bucket is paginated 1000 objects per request, see libs3.get_matching_s3_keys(..), therefore
```

Comment:
@victorskl Did you check if there was a parameter in Boto3 to bundle more paginated objects per request?

Reply:
Yes; 1000 objects is MAX.
 https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.list_objects_v2